indexer/indexer_tests/editable_map_object_test.cpp

```diff
@@ -12,6 +12,32 @@ int8_t GetLangCode(char const * ch)
 {
   return StringUtf8Multilang::GetLangIndex(ch);
 }
+
+struct ExpectedName
+{
+  string m_lang;
+  string m_value;
+};
+
+void CheckExpectations(StringUtf8Multilang const s, vector<ExpectedName> const expectations)
+{
+  size_t counter = 0;
+  s.ForEach([&expectations, &counter](int8_t const code, string const & name) {
+    auto const it = find_if(expectations.begin(), expectations.end(), [&code](ExpectedName const & item)
+    {
+      return GetLangCode(item.m_lang.c_str()) == code;
+    });
+
+    if (it == expectations.end())
+      TEST(false, ("Unexpected language code:", code));
+
+    TEST_EQUAL(name, it->m_value, ());
+    ++counter;
+    return true;
+  });
+
+  TEST_EQUAL(counter, expectations.size(), ());
```

Comment:
It would be nice to print unmet expectations here and the contents of the name in the case of error.